README.md

```diff
@@ -22,10 +22,20 @@ Commit your code regularly and meaningfully. This helps both you (in case you ev
 
 Demonstrate your understanding of this Sprint's concepts by answering the following free-form questions. Edit the `ANSWERS.md` file to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager.
 
-- [ ] Why would you use class component over function components (removing hooks from the question)?
-- [ ] Name three lifecycle methods and their purposes.
-- [ ] What is the purpose of a custom hook?
-- [ ] Why is it important to test our apps?
+- [X] Why would you use class component over function components (removing hooks from the question)?
+If your component needs to maintain state class components can be helpful. Since it maintains it own data with state you can pass state from one component to the next. This can be useful for example if you want to choose for it to update or not.
```

Comment:
This is a good reason to use React components in general because both function and class components can utilize state.

One reason class components could be more useful than functions is because they specifically separate each lifecycle method (`componentDidMount()`, `componentDidUpdate()`, etc), while functions only have `useEffect()`